packages/scss/src/commons/config.scss

```diff
@@ -431,6 +418,44 @@ $palettesInterpolation: (
 	'cc': $cc,
 );
 
+// Tokens : Elevation
+$elevation: (
+	surface: (
+		'sunken': var(--palettes-neutral-50),
+		'default': var(--palettes-neutral-25),
+		'raised': var(--palettes-neutral-0),
+		'backdrop': color.transparentize(var(--palettes-neutral-400), 0.4),
+	),
+	shadow: (
+		'raised':
+			'0 0 0 1px #{color.transparentize(var(--palettes-neutral-400), 0.08)}, 0 1px 2px #{color.transparentize(var(--palettes-neutral-400), 0.4)}, 0 2px 4px #{color.transparentize(var(--palettes-neutral-400), 0.2)}',
+		'overflow':
+			'0 0 0 1px #{color.transparentize(var(--palettes-neutral-400), 0.08)}, 0 0 4px #{color.transparentize(var(--palettes-neutral-400), 0.32)}, 0 0 8px #{color.transparentize(var(--palettes-neutral-400), 0.24)}',
+		'overlay':
+			'0 0 0 1px #{color.transparentize(var(--palettes-neutral-400), 0.08)}, 0 4px 8px #{color.transparentize(var(--palettes-neutral-400), 0.24)}, 0 4px 12px 2px #{color.transparentize(var(--palettes-neutral-400), 0.08)}',
+	),
+);
+
+// Tokens : Colors
+$color: (
```

Comment:
Instead of making map of map, wouldn't it be simpler to make a single level called inputColors?